spec/cohort.spec.js

```diff
@@ -0,0 +1,94 @@
+import Cohort from '../src/cohort.js'
+
+describe('Cohort:', () => {
+  beforeEach(() => {
+    Cohort.resetId()
+  })

```

Comment:
Really cool beforeEach(). Can I ask why you decided to go for a method here instead of just resetting it manually?

Reply:
No particular reason really. `Cohort.lastId = 0` should work here just as well.